loader/src/test/java/net/neoforged/fml/javafmlmod/FMLJavaModLanguageProviderTest.java

```diff
@@ -107,6 +109,40 @@ public EntryPoint(net.neoforged.bus.api.IEventBus modEventBus) {
         assertThat(EVENTS).hasSize(1);
     }
 
+    @Test
+    void testMultipleEntrypoints() throws Exception {
+        installation.setupProductionClient();
+
+        var testJar = installation.writeModJar("test.jar", SimulatedInstallation.createModsToml("testmod", "1.0"));
+        try (var compiler = RuntimeCompiler.create(testJar)) {
+            compiler.builder()
+                .addClass("testmod.EntryPoint", """
+                            @net.neoforged.fml.common.Mod("testmod")
+                            public class EntryPoint {
+                                public EntryPoint() {
+                                    net.neoforged.fml.javafmlmod.FMLJavaModLanguageProviderTest.MESSAGES.add("common");
+                                }
+                            }
+                            """)
+                .addClass("testmod.ClientEntryPoint", """
+                            @net.neoforged.fml.common.Mod(value = "testmod", dist = net.neoforged.api.distmarker.Dist.CLIENT)
+                            public class ClientEntryPoint {
+                                public ClientEntryPoint() {
+                                    net.neoforged.fml.javafmlmod.FMLJavaModLanguageProviderTest.MESSAGES.add("client");
+                                }
+                            }
+                            """)
+                .compile();
+        }
+
+        var result = launch("forgeclient");
+        loadMods(result);
+
+        ModLoader.dispatchParallelEvent("test", Runnable::run, Runnable::run, () -> {}, FMLClientSetupEvent::new);
```

Comment:
This shouldn't be necessary? `loadMods` will already construct the mods I think.

Reply:
Copy+paste moment